agent/command.go

```diff
@@ -202,6 +202,10 @@ func getFunctionName(commandInfo model.PluginCommandConf) string {
 // endTaskSyncCommands returns the commands to sync the task to S3 if it was
 // requested when the task completes.
 func endTaskSyncCommands(tc *taskContext, detail *apimodels.TaskEndDetail) *model.YAMLCommandSet {
+	if tc.taskModel == nil {
+		tc.logger.Task().Error("Error running task sync.")
```

Comment:
Same comment as above.

Reply:
done

Reply:
This doesn't look changed, you may have accidentally forgotten to push the change.